### PR TITLE
remove google analytics

### PR DESCRIPTION
**Pull request by Belco90**
## Changes

Removing Google Analytics since we are not using it at all, but makes the initial load of the app worse.

If at some point we need analytics of the app, we should explore some alternatives to gtag.

## Context

n/a

## Tests

I've checked my work by:

- [ ] Adding new tests or adjusting existing tests
- [X] Testing manually

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,4 @@ cypress.env.json
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+/tsconfig.tsbuildinfo
```
Comment (HonkingGoose) [inline .gitignore:22]: There's a new line in the `.gitignore` is that intentional?

I think this has to do with TypeScript incremental builds?

**Review by HonkingGoose — COMMENTED** [review]
Looks good, but is that change in the `.gitignore` intentional?

File: .gitignore (hunk 1 above)
Comment (Belco90) [reply .gitignore:22]: Yeah, it was on purpose. [This file shouldn't be included in VCS](https://betterstack.dev/blog/lerna-typescript-monorepo/).

Comment (HonkingGoose) [reply .gitignore:22]: Okay! 😄

**Review by HonkingGoose — APPROVED** [review]

**Pull request merged**